cmd/backup-manager/app/backup/backup.go

```diff
@@ -204,3 +281,43 @@ func (bo *Options) brCommandRun(ctx context.Context, fullArgs []string) error {
 	klog.Infof("Run br commond %v for cluster %s successfully", fullArgs, bo)
 	return nil
 }
+
+func (bo *Options) updateProgressFromFile(
+	stopCh <-chan struct{},
+	backup *v1alpha1.Backup,
+	progressFile string,
+	progressStep string,
+	statusUpdater controller.BackupConditionUpdaterInterface,
+) {
+	ticker := time.NewTicker(time.Second)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			data, err := os.ReadFile(progressFile)
+			if err != nil {
+				continue
+			}
+			progressStr := strings.TrimSpace(string(data))
+			progressStr = strings.TrimSuffix(progressStr, "%")
+			if progressStr == "" {
```

Comment:
keep percentage marker and if progressStr="", better to show a 'Progress: 0%'

Reply:
The existing progress field type is `float`.